### PR TITLE
Show a dialog box if there's a geocoder error

### PR DESCRIPTION
**Pull request by rdengate**
For example if the service is down. The error message is similar to what's displayed under Addresses if the GNAF service can't be accessed during the search for a single address.

### CODE REVIEW

**Comment by kring** [comment]
@rdengate mind merging in master?  Conflicts in `CsvCatalogItem` again.

**Comment by rdengate** [comment]
> mind merging in master? Conflicts in CsvCatalogItem again.

Done. This should be good to go, thanks. The errors in Travis are the lint errors that are fixed in a different PR.

File: lib/Models/TableCatalogItem.js [hunk 1]
```diff
@@ -448,10 +448,10 @@ TableCatalogItem.prototype.initializeFromTableStructure = function(tableStructur
                 if (addressGeocoderData.missingAddresses.length > 0) {
                     missingAddressesMessage = "\nThe CSV contains addresses, but " + addressGeocoderData.missingAddresses.length
                         + " can\'t be located on the map:\n"
-                        + addressGeocoderData.missingAddresses.join(", ") + "\n";
+                        + addressGeocoderData.missingAddresses.join(", ") + ".</br></br>";
```
Comment (kring) [inline lib/Models/TableCatalogItem.js]: Should be `<br />`?  Two `\n`s should do the trick too because of the Markdown formatting.

**Comment by kring** [comment]
Can you also update `CHANGES.md`?

**Comment by rdengate** [comment]
@kring done, thanks for reminder.

**Comment by kring** [comment]
Did you see the comment about the `</br>` too?

**Comment by rdengate** [comment]
ah sorry missed that. I will fix it!

**Pull request merged**